content/rosa/terraform/hcp.md

```diff
@@ -37,12 +37,9 @@ This guide will walk you through deploying a ROSA cluster using Terraform.  This
 3. Create a Plan and Apply it
 
     ```bash
-    terraform init
-
-    terraform plan -out tf.plan
-
-    terraform apply tf.plan
-
+    terraform init && \
+      terraform plan -out tf.plan && \
+      terraform apply tf.plan
```

Comment:
Not a blocker, just curious: Why this change in this file but not the other one?